apiserver/facades/client/storage/storage.go

```diff
@@ -143,7 +150,7 @@ func (a *StorageAPI) checkCanWrite() error {
 // storage identified by supplied tags. If specified storage cannot be
 // retrieved, individual error is returned instead of storage information.
 func (a *StorageAPI) StorageDetails(entities params.Entities) (params.StorageDetailsResults, error) {
-	if err := a.checkCanWrite(); err != nil {
+	if err := a.checkCanRead(); err != nil {
```

Comment:
That's an interesting change...

Reply:
I suspect that this was a typo. `StorageDetails` gets invoked when you `juju show-storage` and the call itself does not seem to mutate anything so I thought to make the change to `checkCanRead` instead of adding the "IsSuperUser" check in `checkCanWrite` which gets invoked by other API methods.

Reply:
Fair enough